### ISSUE TITLE
Unable to unlock my two databases since keepassXC upgrade

### ISSUE THREAD

**Issue by Nixeus2** [keepassxreboot/keepassxc#5118]
Hello,

I opened KeepassXC and upgrade to the latest build when KeepassXC prompted me, then, i tried to unlock my two database ( i didn't unlocked since february), i'm 100 sure i use the right password, and KeepassXC told me password was incorrect or HMAC missmatch.

I don't use any keyfile, i just use a password to protect my database.
I'm on Windows 10. Try to uninstall/reinstall KeepassXC, try portable build too...Same issue.

I don't have any backups of these 2 databases.
I tried to create a new database,i can lock/unlock, but still unable to open my two databases :(

Windows 10 x64 ( build 19041.388)

**Comment by droidmonkey** [comment]
Can you open it with the previous version? You can download it from our releases (see top of page)

**Comment by Nixeus2** [comment]
Hello, i tried older version and even KeepassX, not able :(

I'm looking for a brute force programm with a dictionnary file ( i will do the dictionnary fle myself) and hope i will be able to remember the good password :(

Thanks

**Comment by ivanovgeorgi** [comment]
I can confirm exactly the same issue with v.2.6. It couldn't unlock database with no password. 

Once I rolled back to 2.5.4 everything works again.

**Comment by SommerEngineering** [comment]
I have also observed this problem, but can add some details: I have created a new database under Ubuntu linux with version 2.6.0. I also use only a password.

My password is entered by a YubiKey and is therefore always correct. To make this clear: I only use the Yubikey to type the password, not as a hardware key.

Under Ubuntu, unlocking only works in 1 of 4 attempts. So three times I get the error message that the password is wrong and the fourth time it works.

Today I transferred my database to a Windows 10 computer. Here I could not unlock the database at all: After about 25 failed attempts I stopped trying. Unlocking with KeePass did not work either!

Here I suspected that the Yubikey might type the password too fast and KeePassXC, at least under Ubuntu, might swallow a character every now and then. This suspicion was confirmed: I unlocked the database under Ubuntu and changed the password. This time I entered the password manually.

Now the database will be unlocked under Ubuntu immediately (probably it will go wrong from time to time if a character is swallowed). Under Windows 10 I can now unlock the database (with Yubikey). Also with KeePass.

So I guess: The password input field swallows one or more characters every now and then when the password is entered very fast. At least under Ubuntu...

Hope, that helps.
